apps/frontend/src/assets/css/scaffolding.css

```
@@ -32,3 +32,25 @@ body {
 	white-space: nowrap;
 	border: 0;
 }
+
+html {
+	scrollbar-width: thin;
+	scrollbar-color: var(--color-border-primary) var(--color-background-secondary);
+}
+
+html::-webkit-scrollbar {
```

remove html prefix here, we want to apply it for every element
```suggestion
::-webkit-scrollbar {
```